fvtest/compilerunittest/CMakeLists.txt

```diff
@@ -41,6 +41,10 @@ if(OMR_ARCH_POWER)
 	)
 endif()
 
+list(APPEND COMPCGTEST_FILES
```

Comment:
IIRC, `compilerunittest` isn't currently being built on non-Power platforms since it didn't previously contain any tests for those platforms. Now that it does, that needs to be changed.

Reply:
So I guess the change to do is to remove the `if(OMR_ARCH_POWER)` flag in the `CMakeLists.txt `under `fvtest` dir? But this flag should be kept in the `CMakeLists.txt` under `compilerunittest` dir since those three test -`BinaryEncoder.cpp`, `MemoryReferenceExpansion.cpp` and `Peephole.cpp `are for the power platform?